lib/js/components/DatePickers/DatePicker/DatePicker.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+	<div
+		class="ds-datePicker"
+		:class="{ '-ds-box': triggerType === DATE_PICKER_TRIGGER_TYPES.BOX }"
+	>
+		<template v-if="triggerType === DATE_PICKER_TRIGGER_TYPES.TILE">
+			<span
```

Comment:
```suggestion
			<span v-if="label"
```
because it is not a required prop